apps/tenant-management-webapp/src/app/pages/admin/services/script/editor/scriptEditor.tsx

```diff
@@ -65,6 +81,10 @@ export const ScriptEditor: FunctionComponent<ScriptEditorProps> = ({
   const monaco = useMonaco();
   let activeParam = 0;
   let activeSignature = 0;
+
+  useEffect(() => {
+    setScript(selectedScript);
+  }, [selectedScript]);
```

Comment:
The selectedScript is a property. I am not sure what is the reason of detecting a selectedScript here.